src/pages/SingleOperatorPage.tsx

```diff
@@ -360,10 +359,12 @@ export const SingleOperatorPage = () => {
                                     },
                                     {
                                         displayName: 'Staked',
-                                        valueMapper: (element) =>
-                                            `${abbreviateNumber(
-                                                fromAtto(element.amountWei).toNumber(),
-                                            )} ${tokenSymbol}`,
+                                        valueMapper: (element) => (
+                                            <>
+                                                {abbr(fromAtto(element.amountWei))}{' '}
```

Comment:
It'll do numbers too, but w/o `toNumber` it's (1) gonna work, and (2) is so much better!